src/components/content/catalog/services/details/serviceTemplateDetails.tsx

```diff
@@ -0,0 +1,68 @@
+import { CheckCircleOutlined, EnvironmentOutlined, SyncOutlined } from '@ant-design/icons';
+import { Card, Tag } from 'antd';
+import React from 'react';
+import catalogStyles from '../../../../../styles/catalog.module.css';
+import { ServiceTemplateDetailVo } from '../../../../../xpanse-api/generated';
+import { ServicePolicies } from '../policies/ServicePolicies';
+import ServiceDetail from './ServiceDetail';
+import { ServiceHostingOptions } from './ServiceHostingOptions';
+
+function serviceTemplateDetails({
+    isViewDisabled,
+    serviceDetails,
+    groupServiceTemplatesByCsp,
+    serviceCspInQuery,
+    serviceHostingTypeInQuery,
+    onChangeServiceHostingType,
+}: {
+    isViewDisabled: boolean;
+    serviceDetails: ServiceTemplateDetailVo;
+    groupServiceTemplatesByCsp: Map<string, ServiceTemplateDetailVo[]>;
+    serviceCspInQuery: string;
+    serviceHostingTypeInQuery: string;
+    onChangeServiceHostingType: (serviceTemplateDetailVo: ServiceTemplateDetailVo) => void;
+}): React.JSX.Element {
+    return (
+        <>
+            <Card className={catalogStyles.serviceTemplateCard}>
+                <div>
+                    {serviceDetails.isAvailableInCatalog ? (
```

Comment:
currently this tag is not refreshed when the 'unpublish' service is called. Please check if we can reload the data and also show alert. 
We had a similar problem in MyServices and Jinping recently fixed it. Maybe we can use the same solution here as well.